package.json

```diff
@@ -18,24 +18,28 @@
     "url": "https://github.com/icelab/formalist-compose/issues"
   },
   "scripts": {
-    "compile": "./node_modules/.bin/babel --presets ./node_modules/babel-preset-es2015 src --out-dir lib",
+    "compile": "babel --presets babel-preset-es2015 src --out-dir lib",
     "prepublish": "npm run compile",
-    "test": "./node_modules/.bin/babel-node --presets ./node_modules/babel-preset-es2015 test | faucet",
-    "lint": "./node_modules/.bin/eslint 'src/*.js' 'src/**/*.js' 'test/*.js' 'test/**/*.js'",
+    "test": "babel-node --presets babel-preset-es2015 test | faucet",
```

Comment:
@chrisbuttery I did change this to be specific because I was having issues with globally-installed packages conflicting. Is that something that could still happen?